pepper-apis/ddp-common/pom.xml

```diff
@@ -12,7 +12,7 @@
 
     <!-- -->
     <properties>
-        <logback.version>1.4.5</logback.version>
+        <logback.version>1.4.14</logback.version>
```

Comment:
As of now 1.4.14 seem to be the version with no vulnerabilities, so going with it
https://mvnrepository.com/artifact/ch.qos.logback/logback-core